spec/requests/providers/address_lookups_spec.rb

```diff
@@ -64,36 +64,19 @@
       end
     end
 
-    context 'with a valid postcode', :vcr do
+    context 'with a valid postcode' do
```

Comment:
If you've turned of VCR here, does that mean there is a cassette than is no longer used? If so, it needs deleting.

Reply:
I deleted it. But this logic moved to another controller and it created a new VCR cassette.
Git is registering this as moved files : 
![image](https://user-images.githubusercontent.com/482806/49077277-c4de7500-f232-11e8-8074-c29ab630c66f.png)